index.rst

```diff
@@ -1,13 +1,14 @@
-SimpleITK Sphinx Documentation
-==============================
+Welcome to SimpleITK 
+=====================
 
-SimpleITK is a simplified, open source, interface to the National Library
-of Medicine's Insight Segmentation and Registration Toolkit (ITK), a C++
-open source image analysis toolkit which is widely used in academia and
-industry. SimpleITK is available for eight programming languages including
-C++, Python, R, Java, C#, Lua, Ruby, and TCL. Binary distributions of
-SimpleITK are currently available for all three major operating systems
-(Linux, OS X, and Windows).
+SimpleITK is an image-analysis toolkit, providing a simplified programming interface to the algorithms and data structures of the Insight Segmentation and Registration Toolkit. It supports image operations on 2D, 3D and 4D multi-channel images. The toolkit is available in eight programming languages including Python, R, Java, C#, C++, Lua, Ruby, and TCL.
+
+SimpleITK includes several hundred classes providing a broad set of tools required for image analysis. The toolkit provides extensive support for image input and output, supporting more than 20 image file formats. A large number of filtering operations are also available, from basic image analysis operations such as edge detection, binary and grayscale morphological operations, and distance transformations to MRI bias field correction. Additionally, a variety of traditional operators used for image segmentation are available including region growing, connected component labelling, and the watershed transformation. Finally, the toolkit includes a robust registration framework, readily enabling one to align intra and inter modality images, supporting both rigid and non-rigid transformation models.
+
+SimpleITK is widely used by researchers from various domains requiring images analysis capabilities. Amongst others these include analysis of anatomical structures imaged with CT,MR, and PET, and analysis of cellular structures imaged using dual photon microscopy, focused ion beam scanning electron microscopy and focal plane array microscopy. Additionally, the toolkit has been used at a number of academic institutions as a tool for teaching medical image analysis.
```

Comment:
image analysis (singular).

Beginning of the 2nd sentence is awkward and unclear.  Maybe something like 'Applications of SimpleITK have included the analysis of ...'

Or maybe 'These domains include ...'

IMO 'Amongst others' isn't needed, and 'these' needs to refer directly to something.

Reply:
Based on comments and discussion with @blowekamp, PR was completely revised.